CMakeLists.txt

```diff
@@ -232,12 +232,12 @@ endif ()
 # library.
 add_definitions(-DBOOST_TEST_DYN_LINK)
 
-# Detect OpenMP support in a compiler. If the compiler supports OpenMP, flags 
-# to compile with OpenMP are returned and added and the HAS_OPENMP is set to 1. 
+# Detect OpenMP support in a compiler. If the compiler supports OpenMP, flags
+# to compile with OpenMP are returned and added and the HAS_OPENMP is set to 1.
 # This way we can skip calls to functions defined in omp.h with code like:
 # if (HAS_OPENMP == 1) { openMP code here }
 # If OpenMP is found, define HAS_OPENMP to be 1. Otherwise define it to be 0.
-find_package(OpenMP)
+find_package(OpenMP 3)
```

Comment:
It looks like it doesn't. AppVeyor tried to build with OpenMP on when it should have been off...